jerry-core/ecma/builtin-objects/ecma-builtin-string-prototype.inc.h

```diff
@@ -76,6 +76,11 @@ ROUTINE (LIT_MAGIC_STRING_CODE_POINT_AT, ECMA_STRING_PROTOTYPE_CODE_POINT_AT, 1,
 ROUTINE (LIT_MAGIC_STRING_PAD_START,  ECMA_STRING_PROTOTYPE_PAD_START, 2, 1)
 ROUTINE (LIT_MAGIC_STRING_PAD_END,  ECMA_STRING_PROTOTYPE_PAD_END, 2, 1)
 ROUTINE (LIT_GLOBAL_SYMBOL_ITERATOR, ECMA_STRING_PROTOTYPE_ITERATOR, 0, 0)
+
+INTRINSIC_PROPERTY (LIT_MAGIC_STRING_TRIM_START, LIT_INTERNAL_MAGIC_STRING_TRIM_START, ECMA_PROPERTY_FIXED)
+INTRINSIC_PROPERTY (LIT_MAGIC_STRING_TRIM_LEFT, LIT_INTERNAL_MAGIC_STRING_TRIM_START, ECMA_PROPERTY_FIXED)
+INTRINSIC_PROPERTY (LIT_MAGIC_STRING_TRIM_END, LIT_INTERNAL_MAGIC_STRING_TRIM_END, ECMA_PROPERTY_FIXED)
+INTRINSIC_PROPERTY (LIT_MAGIC_STRING_TRIM_RIGHT, LIT_INTERNAL_MAGIC_STRING_TRIM_END, ECMA_PROPERTY_FIXED)
```

Comment:
These routines are not fixed. see `Object.getOwnPropertyDescriptor(String.prototype, "trimLeft")`

Reply:
Already working on this, but thanks for the heads up.